Library/Homebrew/test/pkg_version_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "pkg_version"
+
+describe PkgVersion do
+  specify "#parse" do
```

Comment:
I feel like this could be split into named examples, but I can't think of good names off the top of my head. Thoughts?